lib/types.ts

```diff
@@ -693,6 +698,11 @@ export interface AccessAPI {
     entity: 'ContentType' | ContentType | 'Asset' | 'Entry' | T
   ): Promise<boolean>
   can<T = Object>(action: ArchiveableAction, entity: 'Asset' | 'Entry' | T): Promise<boolean>
+  can<T = Object>(
+    action: 'patch',
+    entity: 'Asset' | 'Entry' | T,
+    patch?: JSONPatchItem[]
```

Comment:
When can `patch` be `undefined`? Doesn't that basically say "check whether you can keep the entity as it is"

Reply:
I think in this case it should be required actually